src/lib/components/Modal.svelte

```diff
@@ -27,7 +27,7 @@
 {#if visible}
   <div
     class="modal"
-    transition:fade
+    transition:fade={{ duration: 150 }}
```

Comment:
Can any of 2 other values be applied here, to have the same ani.time everywhere, or it looks better with 150?
```
  --animation-time-normal: 0.2s;
  --animation-time-short: 0.125s;
  ```

Reply:
Sure 125 works out